src/test/java/ca/qaguru/oranghrmbatch27/pages/PayGradePage.java

```diff
@@ -0,0 +1,45 @@
+package ca.qaguru.oranghrmbatch27.pages;
+
+import ca.qaguru.oranghrmbatch27.library.PageBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class PayGradePage extends PageBase {
+
+    //WebDriver driver;
+
+    public  PayGradePage(WebDriver _driver){
+        super(_driver);
+        //this.driver=_driver;
```

Comment:
Why is this commented out?

Reply:
I dont see the usage of webdriver object in my PayGradePage Class.
I shall remove the comment.